cookbook/src/03-parse.md

```diff
@@ -0,0 +1 @@
+# Parsing Blocks and Transactions
```

Comment:
@tcharding any code snippet up your sleeve that I can deconstruct and explain in simple steps?

Reply:
Not off the top of my head, if I were doing this I'd probably start by grepping the code base for `Transaction::from` looking for hex being parsed then go from there. Same for other types. If you want to add demos of code hitting bitcoin-core you could look at https://github.com/Kixunil/pico-bitcoin-wallet (or my version it `github.com/tcharding/workshop/pico-bitcoin-wallet`).

Reply:
Be warned some the `FromHex` trait has moved out of `hashes` and into `hex-conservative` as of the soon-to-be-released code. If you want to code against `rust-bitcoin v0.30.0` I can do the upgrade easily after we release `v0.31.0` or you can code against `master` and we change it after release to v0.31.0 - what ever is easiest for you.

Reply:
What if we follow `rust-bitcoin#master` and I run the tests CI once a week?
That would make the cookbook most up-to-date with the codebase

Reply:
I think we want to get to a stage where the cookbook is running released code, most users will be using released code. I only mentioned master because we are about to release (soon TM)